src/components/Icon.tsx

```diff
@@ -88,6 +86,8 @@ import {
   WhitepaperIcon,
   WithdrawIcon,
 } from '@/icons';
+import { ChaosLabsIcon } from '@/icons/chaos-labs';
```

Comment:
lmao love a good circular dependency // my brain might be a lil fried but would it resolve the dependency at all if we seperated these two into `@/themedIcons/` or something? (or had them as a separate `ThemeIcon`?) anyways not necessary for PR just a thought

Reply:
yeah i was thinking about this as well - though at any point, a themed icon could be imported into the `dialogs` constants file which would cause this headache all over again. IMO we really need to split out types from values - it will probably help clear up our import tree a good amount